src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/sdk/models/azure-mgmt-containerregistry_enums.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum
+
+
+class ProvisioningState(Enum):
+
+    creating = "Creating"
+    updating = "Updating"
+    deleting = "Deleting"
+    succeeded = "Succeeded"
+    failed = "Failed"
+    canceled = "Canceled"
+
+
+class BuildDefinitionStatus(Enum):
+
+    disabled = "Disabled"
+    enabled = "Enabled"
+
+
+class SourceControlType(Enum):
+
+    github = "Github"
+    visual_studio_team_service = "VisualStudioTeamService"
+
+
+class OsType(Enum):
+
+    windows = "Windows"
+    linux = "Linux"
+
+
+class BuildStatus(Enum):
+
+    queued = "Queued"
+    started = "Started"
```

Comment:
what's the difference between Started and Running?

Reply:
We will address it in next iteration.